src/Loggers/LogText.jl

```diff
@@ -25,6 +25,9 @@ function text_summary(name::String, text::Any)
     #content of the tensor
     textstringval = Vector{Array{UInt8, 1}}()
     if isa(text, AbstractArray)
+        if ndims(text) == 2
+            text = permutedims(text, (2, 1))
+        end
```

Comment:
What is this for?

Reply:
Julia is column major. but tensorboard assumes row major and that creates problems when actually logging matrices as text. after transposing it works as expected.